static/js/authController.ts

```diff
@@ -122,6 +122,20 @@ export class AuthController {
     return this.token ? this.token : null;
   }
 
+  public signInWithGoogle(): void {
+    const googleAuth = new firebase.auth.GoogleAuthProvider();
+    firebase.auth().signInWithPopup(googleAuth).catch((error) => {
+      console.log('Authentication was unsuccessful!');
```

Comment:
Before this gets committed to master we should look through the possible auth errors and make sure we're handling common ones